incuna-sass/mixins/_font-face.sass

```diff
@@ -64,17 +64,15 @@
         // If both IE-8 and woff2 are required output this:
         @if $IE-8 and $woff2
             src: font-url($filepath + '.eot'), font-url($filepath + '.woff2') format('woff2'), font-url($filepath + ".woff") format('woff'), font-url($filepath + ".otf") format('opentype')
+        // Support for just IE8 and lower is required output this:
+        @else if $IE-8 and ($woff2 = false)
```

Comment:
The issue is the same here: `=` is assignment whereas we want `==` for comparison